sites/all/modules/custom/dccr_blocks/dccr_blocks.module

```diff
@@ -49,6 +49,16 @@ function dccr_blocks_block_info() {
     'cache' => DRUPAL_CACHE_GLOBAL,
   );
 
+  $blocks['dccr_final_schedule'] = array(
+    'info' => t('Final Schedule'),
+    'cache' => DRUPAL_CACHE_GLOBAL,
+  );
+
+  $blocks['dccr_my_schedule'] = array(
+    'info' => t('My schedule'),
+    'cache' => DRUPAL_CACHE_PER_ROLE,
```

Comment:
When you change this please test it carefully because when i use this cache feature i had some troubles, use two or more users to verify My schedule functionality